test/unit/factorization/mc/test_compute_t_factor.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "dlaf/common/range2d.h"
 #include "dlaf/communication/communicator_grid.h"
+#include "dlaf/lapack/tile.h"  // workaround for importing lapack.hh
```

Comment:
Why is this a workaround, or rather why is a workaround needed?

Reply:
lapackpp.hh includes complex.h which defines the macro `I` which breaks HPX as I is used in templates.